master/internal/job/job.go

```diff
@@ -200,6 +213,7 @@ func (j *Jobs) Receive(ctx *actor.Context) error {
 				return nil
 			}
 		}
+		ctx.Respond(errors)
```

Comment:
I'm not sure ifthe actor system would recognise this as an `errror` and trigger this. maybe combine the error messages into one?

```
	actorResp := a.m.system.AskAt(job.JobsActorAddr, req)
	if err := actorResp.Error(); err != nil {
		return nil, err
	}
	return resp, nil
```